### PR TITLE
[ja]Translate concepts/storage/projected-volumes into Japanese

### LINKED ISSUE #31432: [ja] Translate concepts/storage/projected-volumes into Japanese
**This is a Feature Request**

**What would you like to be added**

Translate  https://kubernetes.io/docs/concepts/storage/projected-volumes/ into Japanese.

**Why is this needed**

This page has not been translated.

**Comments**

/language ja
/assign

### PR DESCRIPTION
**Pull request by KobayashiD27**
close #31432

### CODE REVIEW

**Comment by jihoon-seo** [comment]
[Netlify build fails](https://app.netlify.com/sites/kubernetes-io-main-staging/deploys/61ea4a704746f000080ce134#L77).

Copying `content/en/examples/pods/storage/projected-secret-downwardapi-configmap.yaml` to `content/ja/examples/pods/storage/projected-secret-downwardapi-configmap.yaml` might resolve the error! 😊

**Comment by KobayashiD27** [comment]
> [Netlify build fails](https://app.netlify.com/sites/kubernetes-io-main-staging/deploys/61ea4a704746f000080ce134#L77).
> 
> Copying `content/en/examples/pods/storage/projected-secret-downwardapi-configmap.yaml` to `content/ja/examples/pods/storage/projected-secret-downwardapi-configmap.yaml` might resolve the error! 😊

Thank you! I forget to stage these files...

File: content/ja/docs/concepts/storage/projected-volumes.md [hunk 1]
```diff
@@ -0,0 +1,91 @@
+---
+title: Projectedボリューム
```
Comment (ptux) [inline content/ja/docs/concepts/storage/projected-volumes.md:2]: Let's translate the term `Projected Volume`  into `投影ボリューム`.

https://eow.alc.co.jp/search?q=projected

Comment (KobayashiD27) [reply content/ja/docs/concepts/storage/projected-volumes.md]: OK, I fix it.

File: content/ja/docs/concepts/storage/projected-volumes.md [hunk 2]
```diff
@@ -0,0 +1,91 @@
...
+ボリュームは、いくつかの既存の`投影`ボリュームソースを同じディレクトリにマップします。
+
+現在、次のタイプのボリュームソースを投影できます。
+
+* [`secret`](/docs/concepts/storage/volumes/#secret)
+* [`downwardAPI`](/docs/concepts/storage/volumes/#downwardapi)
+* [`configMap`](/docs/concepts/storage/volumes/#configmap)
+* `serviceAccountToken`
+
+すべてのソースは、Podと同じnamespaceにある必要があります。詳細は[all-in-one volume designドキュメント](https://github.com/kubernetes/community/blob/master/contributors/design-proposals/node/all-in-one-volume.md)を参照してください。
```
Comment (ptux) [inline content/ja/docs/concepts/storage/projected-volumes.md:24]:
```suggestion
すべてのソースは、Podと同じnamespaceにある必要があります。詳細は[all-in-one volume designドキュメント](https://github.com/kubernetes/design-proposals-archive/blob/main/node/all-in-one-volume.md)を参照してください。
```

File: content/ja/docs/concepts/storage/projected-volumes.md [hunk 3]
```diff
@@ -0,0 +1,91 @@
...
+### 構成例:デフォルト以外のアクセス許可モードが設定されたsecret {#example-configuration-secrets-nondefault-permission-mode}
+
+{{< codenew file="pods/storage/projected-secrets-nondefault-permission-mode.yaml" >}}
+
+各投影ボリュームソースは、specの`sources`にリストされています。パラメータは、2つの例外を除いてほぼ同じです。
+
+* secretについて、ConfigMapの命名と一致するように`secretName`フィールドが`name`に変更されました。
+* `defaultMode`はprojectedレベルでのみ指定でき、各ボリュームソースには指定できません。ただし上に示したように、個々の投影ごとに`mode`を明示的に設定できます。
+
+`TokenRequestProjection`機能が有効になっている場合、現在の[サービスアカウント](/docs/reference/access-authn-authz/authentication/#service-account-tokens)のトークンを指定されたパスのPodに挿入できます。例えば:
```
Comment (ptux) [inline content/ja/docs/concepts/storage/projected-volumes.md:39]:
```suggestion
`TokenRequestProjection`機能が有効になっている場合、現在の[サービスアカウントトークン](/ja/docs/reference/access-authn-authz/authentication/#サービスアカウントトークン)を指定されたパスのPodに挿入できます。例えば:
```

File: content/ja/docs/concepts/storage/projected-volumes.md [hunk 4]
```diff
@@ -0,0 +1,91 @@
...
+`path`フィールドは、投影ボリュームのマウントポイントへの相対パスを指定します。
+
+
+{{< note >}}
+投影ボリュームソースを[`subPath`](/docs/concepts/storage/volumes/#using-subpath)ボリュームマウントとして使用しているコンテナは、それらのボリュームソースの更新を受信しません。
+{{< /note >}}
+
+## SecurityContextの相互作用
+
+[サービスアカウントの投影ボリューム拡張でのファイル権限処理の提案]((https://github.com/kubernetes/enhancements/tree/master/keps/sig-storage/2451-service-account-token-volumes#token-volume-projection))により、正しい所有者権限が設定された投影ファイルが導入されました。
```
Comment (ptux) [inline content/ja/docs/concepts/storage/projected-volumes.md:57]:
```suggestion
[サービスアカウントの投影ボリューム拡張でのファイル権限処理の提案](https://github.com/kubernetes/enhancements/tree/master/keps/sig-storage/2451-service-account-token-volumes#proposal)により、正しい所有者権限が設定された投影ファイルが導入されました。
```

**Review by ptux — CHANGES_REQUESTED** [review]

File: content/ja/docs/concepts/storage/projected-volumes.md [hunk 5]
```diff
@@ -0,0 +1,91 @@
...
+`path`フィールドは、投影ボリュームのマウントポイントへの相対パスを指定します。
+
+
+{{< note >}}
+投影ボリュームソースを[`subPath`](/docs/concepts/storage/volumes/#using-subpath)ボリュームマウントとして使用しているコンテナは、それらのボリュームソースの更新を受信しません。
+{{< /note >}}
+
+## SecurityContextの相互作用
+
+[サービスアカウントの投影ボリューム拡張でのファイル権限処理の提案](https://github.com/kubernetes/enhancements/tree/master/keps/sig-storage/2451-service-account-token-volumes#proposal)により、正しい所有者権限が設定された投影ファイルが導入されました。
```
Comment (ptux) [inline content/ja/docs/concepts/storage/projected-volumes.md:57]: I have made change on the upstream.
https://github.com/kubernetes/website/pull/31496/files

```suggestion
サービスアカウントの投影ボリューム拡張でのファイル権限処理の[提案](https://github.com/kubernetes/enhancements/tree/master/keps/sig-storage/2451-service-account-token-volumes#proposal)により、正しい所有者権限が設定された投影ファイルが導入されました。
```

File: content/ja/docs/concepts/storage/projected-volumes.md [hunk 6]
```diff
@@ -0,0 +1,91 @@
...
+ボリュームは、いくつかの既存の`投影`ボリュームソースを同じディレクトリにマップします。
+
+現在、次のタイプのボリュームソースを投影できます。
+
+* [`secret`](/docs/concepts/storage/volumes/#secret)
+* [`downwardAPI`](/docs/concepts/storage/volumes/#downwardapi)
+* [`configMap`](/docs/concepts/storage/volumes/#configmap)
+* `serviceAccountToken`
+
+すべてのソースは、Podと同じnamespaceにある必要があります。詳細は[all-in-one volume designドキュメント](https://github.com/kubernetes/design-proposals-archive/blob/main/node/all-in-one-volume.md)を参照してください。
```
Comment (ptux) [inline content/ja/docs/concepts/storage/projected-volumes.md:24]: I have made change on the upstream.
https://github.com/kubernetes/website/pull/31496/files

```suggestion
すべてのソースは、Podと同じnamespaceにある必要があります。詳細は[all-in-one volume](https://github.com/kubernetes/design-proposals-archive/blob/main/node/all-in-one-volume.md) デザインドキュメントを参照してください。
```

**Review by ptux — CHANGES_REQUESTED** [review]

File: content/ja/docs/concepts/storage/projected-volumes.md [hunk 7]
```diff
@@ -0,0 +1,91 @@
...
+ボリュームは、いくつかの既存の`投影`ボリュームソースを同じディレクトリにマップします。
+
+現在、次のタイプのボリュームソースを投影できます。
+
+* [`secret`](/docs/concepts/storage/volumes/#secret)
+* [`downwardAPI`](/docs/concepts/storage/volumes/#downwardapi)
+* [`configMap`](/docs/concepts/storage/volumes/#configmap)
+* `serviceAccountToken`
+
+すべてのソースは、Podと同じnamespaceにある必要があります。詳細は[all-in-one volume](https://github.com/kubernetes/design-proposals-archive/blob/main/node/all-in-one-volume.md) デザインドキュメントを参照してください。
```
Comment (nasa9084) [inline content/ja/docs/concepts/storage/projected-volumes.md:24]:
```suggestion
すべてのソースは、Podと同じnamespaceにある必要があります。詳細は[all-in-one volume](https://github.com/kubernetes/design-proposals-archive/blob/main/node/all-in-one-volume.md)デザインドキュメントを参照してください。
```

File: content/ja/docs/concepts/storage/projected-volumes.md [hunk 8]
```diff
@@ -0,0 +1,91 @@
...
+### 構成例:デフォルト以外のアクセス許可モードが設定されたsecret {#example-configuration-secrets-nondefault-permission-mode}
+
+{{< codenew file="pods/storage/projected-secrets-nondefault-permission-mode.yaml" >}}
+
+各投影ボリュームソースは、specの`sources`にリストされています。パラメータは、2つの例外を除いてほぼ同じです。
+
+* secretについて、ConfigMapの命名と一致するように`secretName`フィールドが`name`に変更されました。
+* `defaultMode`はprojectedレベルでのみ指定でき、各ボリュームソースには指定できません。ただし上に示したように、個々の投影ごとに`mode`を明示的に設定できます。
+
+`TokenRequestProjection`機能が有効になっている場合、現在の[サービスアカウントトークン](/ja/docs/reference/access-authn-authz/authentication/#サービスアカウントトークン)を指定されたパスのPodに挿入できます。例えば:
```
Comment (nasa9084) [inline content/ja/docs/concepts/storage/projected-volumes.md:39]: 日本語URLはエンコードされて見た目が良くない＆読みづらくなってしまうので、リンク先に英語でIDを付けて英語IDリンクに直してもらえますか？

File: content/ja/docs/concepts/storage/projected-volumes.md [hunk 9]
```diff
@@ -0,0 +1,91 @@
...
+
+### secret、downwardAPI、およびconfigMapを使用した構成例 {#example-configuration-secret-downwardapi-configmap}
+
+{{< codenew file="pods/storage/projected-secret-downwardapi-configmap.yaml" >}}
+
+### 構成例:デフォルト以外のアクセス許可モードが設定されたsecret {#example-configuration-secrets-nondefault-permission-mode}
+
+{{< codenew file="pods/storage/projected-secrets-nondefault-permission-mode.yaml" >}}
+
+各投影ボリュームソースは、specの`sources`にリストされています。パラメータは、2つの例外を除いてほぼ同じです。
```
Comment (riita10069) [inline content/ja/docs/concepts/storage/projected-volumes.md:34]:
```suggestion
各投影ボリュームソースは、specの`sources`にリストされています。パラメーターは、2つの例外を除いてほぼ同じです。
```

File: content/ja/docs/concepts/storage/projected-volumes.md [hunk 10]
```diff
@@ -0,0 +1,91 @@
...
+
+* secretについて、ConfigMapの命名と一致するように`secretName`フィールドが`name`に変更されました。
+* `defaultMode`はprojectedレベルでのみ指定でき、各ボリュームソースには指定できません。ただし上に示したように、個々の投影ごとに`mode`を明示的に設定できます。
+
+`TokenRequestProjection`機能が有効になっている場合、現在の[サービスアカウントトークン](/ja/docs/reference/access-authn-authz/authentication/#サービスアカウントトークン)を指定されたパスのPodに挿入できます。例えば:
+
+{{< codenew file="pods/storage/projected-service-account-token.yaml" >}}
+
+例のPodには、挿入されたサービスアカウントトークンを含む投影ボリュームがあります。このトークンはPodのコンテナがKubernetes APIサーバーにアクセスするために使用できます。この`audience`フィールドにはトークンのオーディエンスが含まれています。トークンの受信者は、トークンのオーディエンスで指定された識別子で自分自身を識別する必要があります。そうでない場合はトークンを拒否する必要があります。このフィールドはオプションで、デフォルトではAPIサーバーの識別子が指定されます。
```
Comment (riita10069) [inline content/ja/docs/concepts/storage/projected-volumes.md:43]:
```suggestion
この例のPodには、挿入されたサービスアカウントトークンを含む投影ボリュームがあります。このトークンはPodのコンテナがKubernetes APIサーバーにアクセスするために使用できます。この`audience`フィールドにはトークンの受信対象者が含まれています。トークンの受信者は、トークンの`audience`フィールドで指定された識別子で自分自身であるかを識別します。そうでない場合はトークンを拒否します。このフィールドはオプションで、デフォルトではAPIサーバーの識別子が指定されます。
```

File: content/ja/docs/concepts/storage/projected-volumes.md (hunk 8 above)
Comment (KobayashiD27) [reply content/ja/docs/concepts/storage/projected-volumes.md]: ありがとうございます。リンクを修正しました。

**Comment by jihoon-seo** [comment]
@KobayashiD27 If possible, please squash multiple commits into one single commit. 😊

**Comment by kakts** [comment]
Thank you for your contribution.
/lgtm

**Pull request merged**